.github/workflows/deploy-to-ecs.yml

```
@@ -0,0 +1,74 @@
+name: Deploy to ECS
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)